src/almanack/LoC_tracker.py

```diff
@@ -0,0 +1,21 @@
+"""
+This module calculates the absolute value of lines of code (LoC) changed (added or removed)
+in the given Git repository.
+"""
+
+import git
+
+
+def calculate_loc_changes(repo_path: str):
+
+    repo = git.Repo(repo_path)
+
+    total_lines_changed = 0
+
+    for commit in repo.iter_commits():
+        # Retrieve commit statistics
+        diff_stat = commit.stats.total
+        lines_added = diff_stat["insertions"]
```

Comment:
Are you envisioning that you'll do something differently with insertions and deletions in the future? If not, then why not simply append commit.stats.total?

Reply:
This is a good point! I did this initially for print statement testing, however I see how this is unnecessary . This helped me find the "lines" attribute of `stats` which works even better. Thank you for this!